core/deliverservice/deliveryclient.go

```diff
@@ -292,7 +293,10 @@ func (d *deliverServiceImpl) newClient(chainID string, ledgerInfoProvider blocks
 	}
 	backoffPolicy := func(attemptNum int, elapsedTime time.Duration) (time.Duration, bool) {
 		if elapsedTime >= reconnectTotalTimeThreshold {
-			return 0, false
+			if !d.conf.IsStaticLeader {
+				return 0, false
+			}
+			logger.Warning("peer is a static leader, ignoring peer.deliveryclient.reconnectTotalTimeThreshold")
```

Comment:
Just to reiterate the intent of my comments in earlier revisions and our discussion this morning, the idea of having two separate implementations of the policy would be to isolate the two different behaviors we exhibit.